ajax/libs/qwest/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "qwest",
+  "filename": "qwest.min.js",
+  "description": "Ajax library with XHR2, promises and request limit",
+  "version": "4.4.2",
+  "author": "Aurélien Delogu <pyrsmk@dreamysource.fr> (http://dreamysource.fr)",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/pyrsmk/qwest.git"
+  },
+  "keywords": [
+    "ajax",
+    "request",
+    "promises",
+    "xhr"
+  ],
+  "license": "MIT",
+  "homepage": "http://dreamysource.io/project/qwest",
```

Comment:
I don't want to lead users to a 404 page, if it'll be online after months, we can add it then. I think @pyrsmk you can submit a pull request to add it once it's online.